R/drive_download.R

```diff
@@ -23,7 +23,7 @@
 #'   is the file's name on Google Drive and the default location is working
 #'   directory, possibly with an added file extension.
 #' @param type Character. Only consulted if `file` is a native Google file.
-#'   Specifies the desired type of the downloaded file. Will be processed via
+#'   Specifies the desired type of the exported file. Will be processed via
```

Comment:
Good example of how reusing docs via `@inheritParams` means you have to take extra care with word choice. I'm going to use the `@param type` docs in `drive_read()`.